openapi/Swarm.yaml

```
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 
 info:
-  version: 0.5.0
+  version: 0.5.1
```

Please update to 0.5.2
```suggestion
  version: 0.5.2
```